### PR TITLE
Improvements to entropy example

### PR DESCRIPTION
**Pull request by ahojnnes**
Small cleanup after recentrly merged PR.

### CODE REVIEW

File: doc/examples/plot_entropy.py [hunk 1]
```diff
@@ -33,40 +28,47 @@
...
+noise = (noise_mask * np.random.random(noise_mask.shape) - 0.5 *
          noise_mask).astype(np.uint8)
 img = noise + 128
 
-radius = 10
-e = entropy(img, disk(radius))
+entr_img = entropy(img, disk(10))
+
+fig, ax = plt.subplots(1, 3, figsize=(8, 3))
+ax0, ax1, ax2 = ax.ravel()
```
Comment (JDWarner) [inline doc/examples/plot_entropy.py]: Could just use (ax0, ax1, ax2) in prior line. I'm not sure this is a standard for us, but it seems to be common (and the form used later).

Comment (stefanv) [inline doc/examples/plot_entropy.py]: Yes, I'd say that's the more standard form.

**Comment by ahojnnes** [comment]
Should be fixed now. Missed that terrible inconsistency ;-)

**Comment by JDWarner** [comment]
Much improved, thanks @ahojnnes!

**Pull request merged**